template.yaml

```diff
@@ -367,7 +369,7 @@ Resources:
           BY_CRISTIN_ID_INDEX_NAME: !Ref CustomersByCristinIdIndexName
           ID_NAMESPACE: !Sub
             - "https://${Domain}/${Path}"
-            - Domain: !Ref CustomDomain
+            - Domain: !Sub api.${CustomDomain}
```

Comment:
is `api` prefix needed when changing to parameter store?

Reply:
as it is now CustomDomain is used for both the frontend and the backend. This is just to add the api prefix to the CustomDomain. The vallue of the CustomDomain was changed by the new pipeline project